src/OpenTK.Windowing.Desktop/NativeWindow.cs

```diff
@@ -537,11 +537,11 @@ public unsafe Vector2i ClientSize
         /// </remarks>
         public unsafe Vector2i? MinimumSize
         {
-            get => _minimumSize;
+            get => _minimumClientSize;
             set
             {
-                _minimumSize = value;
-                GLFW.SetWindowSizeLimits(WindowPtr, value?.X ?? GLFW.DontCare, value?.Y ?? GLFW.DontCare, _maximumSize?.X ?? GLFW.DontCare, _maximumSize?.Y ?? GLFW.DontCare);
+                _minimumClientSize = value;
+                GLFW.SetWindowSizeLimits(WindowPtr, value?.X ?? GLFW.DontCare, value?.Y ?? GLFW.DontCare, _maximumClientize?.X ?? GLFW.DontCare, _maximumClientize?.Y ?? GLFW.DontCare);
             }
```

Comment:
The documentation for this parameter seems to indicate it's not the minimum client size of the window but instead the minimum size of the window... Which one is correct?

Reply:
I'll do some testing tomorrow, but per the GLFW docs, it's "content size" ... the total size calls have "Frame" in the API names.

https://www.glfw.org/docs/3.3/window_guide.html

> The minimum and maximum size of the content area of a windowed mode window can be enforced with [glfwSetWindowSizeLimits](https://www.glfw.org/docs/3.3/group__window.html#gac314fa6cec7d2d307be9963e2709cc90).

Reply:
Still haven't set up a test yet, but I was just thinking, that verbiage might be a good distinction for OpenTK, too... `FrameSize` and `ClientSize` (or even `ContentSize`).

Reply:
For Windows, at least, min/max is client/content size. I set min to (640,480) and max to (1024,768), then added console output to the Resize event:

```
Resize: Size = (824, 647)
Resize: ClientSize = (800, 600)

Resize: Size = (1048, 815)
Resize: ClientSize = (1024, 768)

Resize: Size = (664, 527)
Resize: ClientSize = (640, 480)
```

I do like `FrameSize` and `ContentSize` better, though, if you'd like me to rename accordingly.

Reply:
https://github.com/opentk/opentk/pull/1680 added the concept of `FramebufferSize` which is distinctly different from `ClientSize` and `Size`. So now we have all of the different values that GLFW exposes. I think it's fine to have `Size`, `ClientSize`, and `FramebufferSize`.